app/src/main/java/io/intrepid/contest/base/BasePresenter.java

```diff
@@ -2,6 +2,7 @@
 
 import android.support.annotation.NonNull;
 
+import io.intrepid.contest.models.Category;
```

Comment:
You probably pulled methods to BasePresenter by mistake and this is leftover code.